src/assets/styles/other/_root-variables.scss

```diff
@@ -14,10 +14,10 @@
   --hover-bg: var(--lines) !important;
   --primary-lighter: var(--primary);
   --font-family-header: 'Titillium Web', sans-serif;
-  --font-family-body: 'IBM Plex Sans', 'Roboto' , 'Helvetica Neue' ,sans-serif;
+  --font-family-body: 'Titillium Web', 'Roboto', 'Helvetica Neue', sans-serif;
```

Comment:
This should be `IBM Plex Sans`

We don't want body text to be `Titillium Web`. It is too stylized and not consistent with our design system